tests/Test/AbstractFixerTestCase.php

```diff
@@ -322,6 +322,26 @@ final public function testDeprecatedFixersHaveCorrectSummary(): void
         }
     }
 
+    final public function testDeprecatedFixersDoNotHaveDeprecatedSuccessor(): void
+    {
+        if (!$this->fixer instanceof DeprecatedFixerInterface || [] === $this->fixer->getSuccessorsNames()) {
+            $this->addToAssertionCount(1);
+
+            return;
+        }
+
+        foreach ($this->fixer->getSuccessorsNames() as $successorsName) {
+            self::assertFalse(
+                self::getFixerByName($successorsName) instanceof DeprecatedFixerInterface,
+                sprintf(
+                    'Successor fixer `%s` for deprecated fixer `%s` is deprecated itself.',
+                    $successorsName,
```

Comment:
```suggestion
        foreach ($this->fixer->getSuccessorsNames() as $successorName) {
            self::assertFalse(
                self::getFixerByName($successorName) instanceof DeprecatedFixerInterface,
                sprintf(
                    'Successor fixer `%s` for deprecated fixer `%s` is deprecated itself.',
                    $successorName,
```

It could be interpreted as `successor's name`, but IMHO singular form is better here.

Reply:
Autosuggestion failed this time and I did not notice.